test/Streamly/Test/Data/Parser/ParserD.hs

```diff
@@ -262,6 +262,23 @@ lookAhead =
                         where
                             list_length = Prelude.length ls
 
+sliceBeginWith :: Property
+sliceBeginWith =
```

Comment:
The same comments made above apply here as well.

Reply:
I think I should remove this test from here as mentioned here(https://github.com/composewell/streamly/pull/839#discussion_r549729005)